src/components/App.js

```diff
@@ -49,8 +60,10 @@ const App = () => {
             <Todo
               key={index}
               index={index}
-              todo={todo}
-              handleDeleteIconClick={handleDeleteIconClick}
+              title={todo.title}
+              onDelete={handleDeleteIconClick}
```

Comment:
Como cambiamos el _event listener_, el _handler_ debe corresponderse con el nombre, dentro de `App` ya no tenemos acceso al `DeleteIcon`, esto pertenece a `Todo` ahora.

```suggestion
              onDelete={handleTodoDelete}
```